Header header
MotorState[] motorStates
string command

